core/vm/contracts.go

```diff
@@ -1048,3 +1070,66 @@ func (c *bls12381MapG2) Run(input []byte) ([]byte, error) {
 	// Encode the G2 point to 256 bytes
 	return g.EncodePoint(r), nil
 }
+
+// pointEvaluation implements the EIP-4844 point evaluation precompile.
+type pointEvaluation struct{}
```

Comment:
Can we perhaps give this a more meaningful name throughout the code? It doesn't say much.

e.g. kzgPointEvaluation or something